src/com/dre/brewery/recipe/SimpleItem.java

```diff
@@ -148,5 +148,12 @@ public static void registerItemLoader(P p) {
 		p.registerForItemLoader("SI", SimpleItem::loadFrom);
 	}
 
+	@Override
+	public String displayName() {
+		// approximate a user-friendly display name since
+		// server doesn't have access to translations :/
+		return mat.toString().toLowerCase().replace("_", " ");
```

Comment:
Using Paper it is possible to use `TranslatableComponent` in the Lore. If the Server is running Paper those could be used to display the proper item name